src/exampleTypes/ResultTable.js

```diff
@@ -41,6 +41,84 @@ let popoverStyle = {
   userSelect: 'none',
 }
 
+let HighlightedColumn = withStateLens({ viewModal: false })(
+  observer(
+    ({
+      visibleFields,
+      node,
+      highlight = _.result('slice.0.highlight', getResults(node)),
+      header,
+      Cell = header ? 'th' : 'td',
+      Table = 'table',
+      Modal = null,
+      viewModal,
+    }) => {
+      let alsoHighlighted = _.difference(_.keys(highlight), visibleFields)
+      let getConnector = i => {
+        let length = alsoHighlighted.length
+        return i === length - 1 ? '' : i === length - 2 ? ' and ' : ', '
+      }
+      return highlight && !_.isEmpty(alsoHighlighted) ? (
+        <Cell key="highlight">
+          {Modal && (
+            <Modal isOpen={viewModal}>
+              <h3>Other Matching Fields</h3>
+              <Table>
+                <thead>
+                  <tr>
+                    {_.map(
+                      x => (
+                        <th key={x}>{_.startCase(x)}</th>
```

Comment:
We actually should check the schemas in case there are labels for those fields. Just because the fields aren't in the table currently, it doesn't mean that they're not still on the schemas